orca/service_test.go

```diff
@@ -126,7 +145,7 @@ func (s) TestE2E_CustomBackendMetrics_OutOfBand(t *testing.T) {
 				errCh <- fmt.Errorf("UnaryCall failed: %v", err)
 				return
 			}
-			time.Sleep(time.Millisecond)
+			time.Sleep(10 * time.Millisecond)
```

Comment:
why increase the sleep time? if you stick the mutex approach, we will make sure all 20 rpc calls are serialized